### PR TITLE
Fix compile errors in mono-service

### PR DESCRIPTION
**Pull request by kimbor**
Fix compile errors in hedera-mono-service following latest pbj updates.

**Related issue(s)**:

Issue #5833

### CODE REVIEW

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java [hunk 1]
```diff
@@ -80,7 +80,11 @@ void returnsExpectedAccountId() {
         given(seqNo.getAndIncrement()).willReturn(555L);
 
         // when:
-        AccountID newId = subject.newAccountId(sponsor);
+        AccountID newId = AccountID.newBuilder()
+                .setRealmNum(sponsor.getRealmNum())
+                .setShardNum(sponsor.getShardNum())
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java:85]: nit: This should be 
```
AccountID.newBuilder()
                .setRealmNum(0)
                .setShardNum(0)
```
Should not matter for the test.

**Review by Neeharika-Sompalli — APPROVED** [review]

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java (hunk 1 above)
Comment (kimbor) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java:85]: The change that you suggested makes the test fail on line 90 `        assertEquals(asAccount("1.2.555"), newId);`

Comment (mhess-swl) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java:85]: Agreed, after our conversation last friday `.setRealmNum(0)` and `.setShardNum(0)` are what I thought we were going with (and the test would be changed to "0.0.555"). Someone remember something different?

Comment (kimbor) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java:85]: That sounds right. I will update the test to make it work.

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**